sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py

```diff
@@ -89,6 +90,41 @@ def test_torch_run_inference_imagenet_mobilenetv2(self):
       filename, prediction = prediction.split(',')
       self.assertEqual(_EXPECTED_OUTPUTS[filename], prediction)
 
+  @pytest.mark.uses_pytorch
+  @pytest.mark.it_postcommit
+  def test_torch_run_inference_bert_for_masked_lm(self):
+    test_pipeline = TestPipeline(is_integration_test=True)
+    # Path to text file containing some sentences
+    file_of_sentences = 'gs://apache-beam-ml/datasets/custom/sentences.txt'  # disable: line-too-long
+    output_file_dir = 'gs://apache-beam-ml/testing/predictions'
```

Comment:
For test output, it's better to use a bucket with a lifecycle configured to leave less clutter behind, for example:

:~$ gsutil lifecycle get gs://temp-storage-for-end-to-end-tests/
{"rule": [{"action": {"type": "Delete"}, "condition": {"age": 14}}]}

Lifecycle may be per bucket (not sure if we can configure it just for `./testing/predictions`), so switching outputs for all tests to gs://temp-storage-for-end-to-end-tests/ may be easiest.

cc: @AnandInguva FYI.

Reply:
SG, thanks.